apps/src/templates/amazonFutureEngineerEligibility/amazonFutureEngineerEligibilityForm.jsx

```diff
@@ -273,6 +316,43 @@ export default class AmazonFutureEngineerEligibilityForm extends React.Component
                     : null
                 }
               />
+              <p>Tell CSTA a little bit about yourself:</p>
+              <label
+                style={styles.descriptiveText}
+                htmlFor="professionalRoleSelect"
+              >
+                What is your current role?
+              </label>
+              <select
+                style={styles.dropdownPadding}
+                id="professionalRoleSelect"
+                name="professionalRole"
+                value={this.state.professionalRole}
+                onChange={this.handleRoleChange}
+              >
+                {CSTA_PROFESSIONAL_ROLES.map(role => (
+                  <option value={role} key={role}>
+                    {role}
+                  </option>
+                ))}
+              </select>
+              <fieldset className="gradebands-group">
+                <legend style={styles.descriptiveText}>
+                  What grade bands do you teach?
+                </legend>
+                {CSTA_GRADE_BANDS.map((grade, index) => (
+                  <Checkbox
+                    style={styles.checkboxItem}
+                    key={index}
+                    checked={this.state.gradeBands[index]}
+                    onChange={() => this.handleMultiSelect(index)}
+                  >
+                    <label style={styles.checkboxLabel} htmlFor={grade}>
```

Comment:
For instance, tmlc (on twitter, tmlt means to my last tweet and I want to start using it to refer to comments), it's hard for me to tell which of these need to be labels vs aria-labels!